x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -105,6 +105,66 @@ describe('<IndexManagementHome />', () => {
     });
   });
 
+  describe('toggle system indices', () => {
+    beforeAll(async () => {
+      const systemIndex = {
+        health: 'green',
+        status: 'open',
+        name: 'testSystemIndex',
+        uuid: 'AToRD7lATjKLDV3BYvywtQ',
+        primary: '1',
+        replica: '0',
+        documents: '3',
+        size: '17.7kb',
+        isFrozen: false,
+        aliases: [],
+        hidden: true,
+        ilm: {},
+        isRollupIndex: false,
+        isFollowerIndex: false,
+      };
+
+      httpRequestsMockHelpers.setLoadIndicesResponse([systemIndex]);
+
+      testBed = await setup();
+      const { component } = testBed;
+
+      await act(async () => {
+        await nextTick();
+        component.update();
```

Comment:
I think our current understanding of flaky tests is that we don't need to use `nextTick()` instead I think we put the `setup()` in an `act()` wrapper, similar to this: 
```
await act(async () => {
    testBed = await setup();
});
const { component } = testBed;
component.update();
```
The code in Index Management is a bit older than in ILM, that is why there are still `nextTcik()` calls. But could you please update your new tests with this snippet?

Reply:
Yes, please please never use `nextTick()` anymore 👍